source/app/components/DocumentActionBar/DocumentActionBar.js

```diff
@@ -31,26 +33,55 @@ const DocumentActionBar = ({ document, isComplianceTrack, redactMatches }) => {
         {isComplianceTrack && (
           <>
             <Button
-              onClick={() => dispatch(clearRedactions(documentId))}
-              disabled={!getAreRedactionsOnDocument(document)}
+              onClick={() => dispatch(saveRedactions(documentId, redactions))}
+              isLoading={isSavingRedactions}
+              disabled={!areUnsavedRedactions}
               size='sm'
               bg='#eee'
               border='1px solid #cfcfcf'
               borderRadius='2px'
-              fontSize='1rem'
+              fontSize='0.9rem'
+              fontWeight='normal'
             >
-              Clear Redactions
+              Save
             </Button>
 
             <ExclusionListMenu document={document} />
 
             <ExportPreview />
+
+            <Button
```

Comment:
Seems the Export and Save button both have similar styles. Should we pull this as a separate component?

Reply:
Done